cmd/deploy/deploy.go

```diff
@@ -274,6 +270,39 @@ func deployApp(ctx context.Context, appVersionOutput app.CreateAppVersionOutput,
 	return nil
 }
 
+type statusUpdater struct {
+	verbose               bool
+	lastStatus            *string
+	sameStatusUpdateCount int
```

Comment:
Can be `uint` since it is always non-negative.

Reply:
Fixed in 1d22137ce6d2b61c11d3471d2caa7aa724490d10